modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -256,7 +258,7 @@ This name is part of the domain of your deployed app. For example, naming your a
 If not set, by default it is the location of the JAR generated at the package phase
 | environment | The CloudHub environment to which you want to deploy.
 | properties | Top-Level element. +
-If you need to set properties for your Mule application being deployed, you can use the <properties> top-level element:
+If you need to set properties for the Mule application you are deploying, you can use the <properties> top-level element:
```

Comment:
another small style thing: backticks around <properties> so that it's `<properties>`